ui/components/app/confirm/footer/footer.tsx

```diff
@@ -1,5 +1,40 @@
 import React from 'react';
+import {
+  Box,
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '../../../component-library';
+import {
+  BlockSize,
+  Display,
+  FlexDirection,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
 
-const Footer = () => <div>CONFIRM FOOTER COMPONENT</div>;
+const Footer = () => {
+  const t = useI18nContext();
+  return (
+    <Box
+      display={Display.Flex}
+      flexDirection={FlexDirection.Row}
+      gap={4}
+      padding={4}
+      width={BlockSize.Full}
+    >
+      <Button
+        block
+        data-testid="cancel-button"
```

Comment:
maybe we can increase the specificity by mapping the file path to the name here
```suggestion
        data-testid="confirm-footer-cancel-button"
```